migrations/sql/V6__Add_file_format_table.sql

```diff
@@ -1,5 +1,5 @@
 CREATE TABLE file_format (
```

Comment:
Should this be a new migration? We've now run up to migration 8 on dev, so we can't go back and edit old migrations without wiping out the table or rolling back (which I've just discovered isn't supported by Flyway community edition :scream:).